HandheldCompanion/Controls/GameView.xaml

```diff
@@ -0,0 +1,293 @@
+﻿<UserControl
+    x:Class="HandheldCompanion.Controls.GameView"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:converters="clr-namespace:HandheldCompanion.Converters"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:gif="http://wpfanimatedgif.codeplex.com"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:viewmodels="clr-namespace:HandheldCompanion.ViewModels"
+    xmlns:local="clr-namespace:HandheldCompanion.ViewModels"
+    xmlns:ui="http://schemas.inkore.net/lib/ui/wpf/modern"
+    d:DataContext="{d:DesignInstance Type=viewmodels:GameViewModel}"
+    d:DesignHeight="1200"
+    d:DesignWidth="640"
+    
+    FontSize="16"
+    Foreground="{StaticResource PrimaryForegroundColor}"
+    mc:Ignorable="d">
+
+
+
+    <UserControl.Resources>
+        <BooleanToVisibilityConverter x:Key="BooleanToVisibilityConverter" />
+        <converters:ImageToImageSourceConverter x:Key="ImageToImageSourceConverter" />
+        <converters:IconToImageSourceConverter x:Key="IconToImageSourceConverter" />
+
+        <Style TargetType="TextBox">
```

Comment:
Not a fan of this stuff. I'd rather you give it a x:Key and assign it to the specific ones.
Also put it in a ResourceDictionary